documentation/api/definitions.md

```diff
@@ -297,6 +299,12 @@ Detailed information about an operation.
     "status": "Succeeded",
     "egressProviderName": "monitorBlob",
     "isStoppable": false,
+    "process": {
```

Comment:
Note that this wasn't changed in this PR - I just noticed that this was missing while reviewing my own change.